workbench-core/authorization/src/dynamicAuthorization/wbcGroupManagementPlugin.test.ts

```diff
@@ -530,6 +530,14 @@ describe('WBCGroupManagemntPlugin', () => {
       );
     });
 
+    it('throws ForbiddenError when status tried to go from deleted to delete_pending', async () => {
```

Comment:
we'll want tests for throwing `ForbiddenError` when:
```
active -> deleted
delete_pending -> active
deleted -> active
deleted -> delete_pending
(not sure if the below are necessary since they are no-ops)
active -> active
delete_pending -> delete_pending
deleted -> deleted
```

Reply:
I'm going to resolve this since `deleted` is now not valid state transition and `active` -> `deleted`
`delete_pending` -> `active` are covered.